formspree/routes.py

```diff
@@ -30,6 +30,8 @@ def configure_routes(app):
     app.add_url_rule('/forms', 'forms', view_func=forms.views.forms, methods=['GET', 'POST'])
     app.add_url_rule('/forms/<hashid>/', 'form-submissions', view_func=forms.views.form_submissions, methods=['GET'])
     app.add_url_rule('/forms/<hashid>.<format>', 'form-submissions', view_func=forms.views.form_submissions, methods=['GET'])
+    app.add_url_rule('/forms/<hashid>/delete', 'form-deletion', view_func=forms.views.form_deletion, methods=['GET'])
+    app.add_url_rule('/forms/<hashid>/<submissionid>/delete', 'submission-deletion', view_func=forms.views.submission_deletion, methods=['GET'])
```

Comment:
I don't think we should actually delete the form. It should be marked as deleted and then displayed in a (possibly collapsed) section below the active forms. Because the form is never really deleted, it shouldn't be necessary to provide a confirmation.